db/migrate/20231102062958_add_has_prior_last_names_to_state_file_az_intake.rb

```diff
@@ -0,0 +1,5 @@
+class AddHasPriorLastNamesToStateFileAzIntake < ActiveRecord::Migration[7.1]
+  def change
+    add_column :state_file_az_intakes, :has_prior_last_names, :integer, default: 0, null: false
```

Comment:
could technically live without this being in the database since we have the prior_last_names field already but it makes it a lot easier to have the right state if someone goes back in the flow to edit this page

Reply:
agreed, I tried to go without the extra enum column in another area and regretted it